### ISSUE TITLE
Conflict between your dependencies and Distriqt dependencies

### ISSUE THREAD

**Issue by obuw** [myflashlab/common-dependencies-ANE#43]
Hi,

I have just purchased the UDID ANE, but I am getting conflicts when I try to use it alongside the NativeMaps ANE from Distriqt.

Since you don't have a native maps ane that I know of, I have no choice but to use theirs.

What can I do to somehow merge the androidSupport ANE's from both providers and have them work together?

Furthermore, why does a simple UDID ANE have *9* different dependencies? I don't want to access telephony services, advertiser id's, or any of that other stuff. I just simply want an android id / vendor id.

I'd appreciate it if you can provide a solution. Otherwise, I will have to ask for a refund, as the ANE is unusable in its current state.

Thank you.

**Comment by myflashlab** [comment]
We can't know what dependencies are included in other ANE-providers ANEs. but when there are conflicts, you may try not including the dependencies to see which one is making the trouble. If the referenced dependency is available on the other provider's ANEs, then the UDID ANE might also work fine by reading the dependencies from there.

**Comment by obuw** [comment]
I tried every combination of not including dependencies from each provider, but nothing worked. I think since the other provider is also an extremely popular one like yours, it may be in your best interest to try to have a version of your core dependencies that are compatible with theirs. Or perhaps a lightweight version of the ANE without so many dependencies.

At the end of the day, I found a simple free ANE from someone else that does the same thing, with no dependencies or conflicts, at 1/5 the file size.
